pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -661,6 +665,10 @@ self: super: {
     };
   });
 
+  venn-nvim = super.venn-nvim.overrideAttrs (old: {
```

Comment:
There is no Makefile in the repo https://github.com/jbyuki/venn.nvim/

Reply:
Thanks! Good point. Fixed. :)